tensorflow/python/platform/sysconfig.py

```diff
@@ -51,5 +53,32 @@ def get_lib():
   import tensorflow as tf
   return _os_path.join(_os_path.dirname(tf.__file__))
 
+
+def get_compile_flags():
+  """Get the compilation flags for custom operators.
+
+  Returns:
+    The compilation flags.
+  """
+  import tensorflow as tf
+  flags = []
+  flags.append('-I%s' % get_include())
```

Comment:
These are gcc flags. It's not clear what to do on windows, since you can use MSVC and gcc-based compilers. I think we should assume MSVC-style flags on windows.

Reply:
Is it OK to just check `platform.system() == 'Windows'`, and if so output MSVC-style flags?

Reply:
So, I think that will do for now.

Reply:
Gotcha.  I think includes (/I) are pretty straightforward.  Is there any instruction that specifies which link options are necessary for Windows?  https://www.tensorflow.org/extend/adding_an_op#compile_the_op_using_your_system_compiler_tensorflow_binary_installation port for Windows.